app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/utils/BeaconPlacer.kt

```diff
@@ -0,0 +1,98 @@
+package ch.epfl.cs311.wanderwave.ui.components.utils
```

Comment:
I don't think this should go there, I agree it's an utils file, but not at all for ui only. This seems much more like a background that the user may not even be aware or a little aware.